c_cxx/QNN_EP/mobilenetv2_classification/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+project(qnn_ep_sample C CXX)
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+cmake_minimum_required(VERSION 3.13)
+
+option(ONNXRUNTIME_ROOTDIR "onnxruntime root dir")
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include_directories(${ONNXRUNTIME_ROOTDIR}/include/onnxruntime/core/session/)
+
+  
+ADD_EXECUTABLE(qnn_ep_sample  main.cpp)
+if(HAVE_TENSORRT_PROVIDER_FACTORY_H)
+  target_compile_definitions(qnn_ep_sample PRIVATE -DHAVE_TENSORRT_PROVIDER_FACTORY_H)
+endif()
```

Comment:
Can remove this

Reply:
no, it's required.

---
In reply to: [1203258511](https://github.com/microsoft/onnxruntime-inference-examples/pull/246#discussion_r1203258511) [](ancestors = 1203258511)

Reply:
This should only be required for TensorRT, right? Why is this needed here?

Reply:
For example, this is only needed for examples that need to access the tensorrt provider option struct. Ex:
https://github.com/microsoft/onnxruntime-inference-examples/blob/1cc89a5cc5475ade2858f140d7b40316421ec932/c_cxx/squeezenet/main.cpp#L11

Reply:
I see. I saw the comments on line 20 which is endif().

---
In reply to: [1203275011](https://github.com/microsoft/onnxruntime-inference-examples/pull/246#discussion_r1203275011) [](ancestors = 1203275011)